File: Task3/C/Subtask2/Task2.c
#include <stdio.h>
#include <string.h>

int main()
{
	FILE *inpfile,*opfile ;
	char dat[100];

	inpfile=fopen("input.txt","r");
	opfile=fopen("output.txt","w");

	while (fgets(dat,sizeof(dat),inpfile !=NULL){
	fputs(dat,opfile);
	}
	fclose(inpfile);
	fclose(opfile);
	return 0;
}

File: Task3/java/Subtask2/Task2.java
import java.io.FileReader;
import java.io.FileWriter;
import java.io.BufferedReader;
import java.io.BufferedWriter;
import java.io.IOException;

public class Filein{
    String inputFilepath="input.txt";
    String outputFilePath="output.txt";
    
}
    try{
        FileReader fr= new FileReader('input.txt');
        FileWriter fw= new FileWriter('output.txt');

        int dat;
        while ((dat=fr.read())!=-1){
            fw.write(dat);
        }
        
        fr.close()
        fw.close()
    

    }
